src/Store.js

```diff
@@ -176,113 +237,117 @@ class Store {
    *
    * @method findOne
    * @param {String} type the type to find
-   * @param id
-   * @param {Object} options
+   * @param {String} id the id of the record to find
+   * @param {Object} options { fromServer, queryParams }
+   * @return {Object} record
    */
   findOne = (type, id, options = {}) => {
-    const { fromServer, queryParams } = options
+    const { fromServer } = options
```

Comment:
I'm not sure what you mean by this. I changed the method signatures for all of the methods we'll be calling directly from portal to just pass the options (queryParams being one of those possible options - but not the only one). I found it confusing when it was inconsistent.

Reply:
I've actually decided to break this work up. I have a new PR with all of these same changes but it won't always return promises from `find`, it still returns promises or records depending on how it found the records.It will still be a minor version bump because we remove the relationship functionality so that's still a breaking change technically. The new PR is here (I updated based on comments on this PR too) https://github.com/artemis-ag/mobx-async-store/pull/88

My idea is:
1. First PR: Store methods will have exact same functionality as they've always had - but there will be `fromServer` deprecation warnings, new methods to get and fetch with, and the find methods will use these new get/fetch methods.

2. Second PRs: 
a. Update `findAll` to return Promise if fromServer option is not there but records were fetched from the store. (Right now this is the only way it doesn't return a promise) 
b. Update portal if this breaks any calls to `findAll`
c. Update mobile if this breaks any calls to `findAll`

3. Third PRs:
a. Update `findMany` to return a Promise even if  the records were fetched form the store. 
b. Update portal if this breaks any calls to `findMany`
c. Update mobile if this breaks any calls `findMany`

4. Fourth PRs:
a. Update `findOne` to return a Promise even if the records were fetched from the store.
b. Update portal if this breaks any calls to `findOne`
c. Update mobile if this breaks any call to `findOne`